servlet/src/main/java/com/linecorp/armeria/server/servlet/DefaultServletInputStream.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.server.servlet;
+
+import static java.util.Objects.requireNonNull;
+
+import java.io.IOException;
+
+import javax.servlet.ReadListener;
+import javax.servlet.ServletInputStream;
+
+import io.netty.buffer.ByteBuf;
+
+/**
+ * The servlet input stream.
+ */
+final class DefaultServletInputStream extends ServletInputStream {
+    private final ByteBuf source;
+
+    /**
+     * Creates a new instance.
+     */
+    DefaultServletInputStream(ByteBuf source) {
+        requireNonNull(source, "source");
+        this.source = source;
+    }
+
+    @Override
+    public int readLine(byte[] b, int off, int len) throws IOException {
+        throw new UnsupportedOperationException("Not supported yet.");
```

Comment:
No, because all request data parsed to servlet `parameters`. I think `readLine()` is only useful to read a large size upload data. And we don't support now.

Reply:
If we support `read()`, shouldn't we support `readLine()` as well?

Reply:
I think, we also shouldn't support `read()` now.

Reply:
Then, is it better to just not support in `DefaultServletHttpRequest.getInputStream()`?
Is there any chance that content type of the request is not FORM_DATA so that the content is used directly by the user?

Reply:
I have thought about your question before. But I have not found a solution for this problem. Does Armeria provide a API return InputStream to User without aggregate operation?

Reply:
>  But I have not found a solution for this problem. 

I might lost a context. 😄 What was the problem you are mentioning?

> Yes, I think so.

If so, let's do it. 😉

Reply:
> I might lose a context. 😄 What was the problem you are mentioning?
I mean: Input stream support read directly bytes data, and we can read We can read each part of data. But if we aggregated request, I think all byte already read to byte array. Therefore use Input stream is unnecessary